### ISSUE TITLE
Exception: in_array() [...] must be of type array, int given in file [...]

### ISSUE THREAD

**Issue by miracuruzrh** [pierre-alain-b/rainloop-nextcloud#275]
Hi,

I would like to install the rainloop app but when I click on activate there is a message: "An error occured during the request. Unable to proceed."

When I look in the log from nextcloud I see this messages:
`
Exception: in_array(): Argument #2 ($haystack) must be of type array, int given in file '/var/www/vhosts/boonstra.ch/httpdocs/nextcloud/lib/private/Installer.php' line 128
`
My nextcloud installation is on a managed webhoster. So I have no SSH access to my nextcloud.
I'm running nextcloud v23.0.3

If anyone have an idea how to resolve this error, that would be much apprechiated.
Plese let me know if you need further information.

Best regards
Simon

**Comment by pierre-alain-b** [comment]
The error seeems to refer to a nextcloud file... do other apps install seamlessly?

**Comment by miracuruzrh** [comment]
Hi Pierre

Thanks for having a look. I just tried another app and it failed as well.
Now I have installed nextcloud on my raspberry to check how it performs with a fresh installation and fresh database and there it worked. Therefore I can confirm that it is instance-specific. Then I think I have to search where's an issue with my nextcloud installation then.

Indeed I have also a problem with background jobs at the moment. Webcron is active all 5min. but still I've get a message that some jobs are overdue. I don't know if it's related to this problem. I think I try to resolve first the issue with the background jobs and then try to repair the installation to be able to install apps again.

I have already erased the nextcloud installation and used the same mariadb which I already have. Maybe the issue is there.
